### PR TITLE
Climate range slider

### PR DESCRIPTION
**Pull request by justweb1**
- Fix hard-coded styling for size.
- Fix styling for colors temp-slider and temp-range-slider

### CODE REVIEW

File: src/more-infos/more-info-climate.html [hunk 1]
```diff
@@ -50,11 +50,20 @@
         width: 100%;
       }
 
-      paper-range-slider {
-        width: 198px;
-        height: 2px;
-        margin-bottom: 16px;
-        --paper-range-slider-active-color: var(--primary-color);
+      .heat {
+        --paper-slider-active-color: var(--paper-orange-400);
```
Comment (balloob) [inline src/more-infos/more-info-climate.html]: These colors do not exist in the production build. [Available colors](https://github.com/home-assistant/home-assistant-polymer/blob/master/src/resources/ha-style.html)

File: src/more-infos/more-info-climate.html [hunk 2]
```diff
@@ -74,8 +83,10 @@
           <div hidden$='[[computeTargetTempHidden(stateObj)]]'>Target Temperature</div>
           <paper-slider
             id="temp-slider"
+            class$='[[stateObj.attributes.operation_mode]]'
```
Comment (balloob) [inline src/more-infos/more-info-climate.html]: Please add the class to the parent div. You don't know how web components handle class.

**Review by balloob — CHANGES_REQUESTED** [review]

File: src/more-infos/more-info-climate.html (hunk 1 above)
Comment (justweb1) [inline src/more-infos/more-info-climate.html]: Can I add them?

Comment (justweb1) [inline src/more-infos/more-info-climate.html]: It looks like the green is already there. I can just add the blue and orange.

File: src/more-infos/more-info-climate.html (hunk 2 above)
Comment (justweb1) [inline src/more-infos/more-info-climate.html]: When I did this it broke it... Let me see if I can fix it on the parent div.

File: src/more-infos/more-info-climate.html (hunk 1 above)
Comment (justweb1) [inline src/more-infos/more-info-climate.html]: fixed https://github.com/home-assistant/home-assistant-polymer/pull/111/commits/0c3b45ba95d018a0143a33681bf2053881372186

File: src/more-infos/more-info-climate.html (hunk 2 above)
Comment (justweb1) [inline src/more-infos/more-info-climate.html]: Fixed https://github.com/home-assistant/home-assistant-polymer/pull/111/commits/c1951c3306546d3f43f0d5d46baeba4e5efb8007

**Review by balloob — APPROVED** [review]

**Pull request merged**